examples/with-universal-configuration-runtime/pages/index.js

```diff
@@ -1,17 +1,19 @@
 import React from 'react'
-import env from '../lib/env'
+import getConfig from 'next/config'
```

Comment:
it seems like 2 PRs are mixed here

Reply:
I resolved that, now they are seperated...